client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/style.scss

```diff
@@ -144,7 +144,7 @@
 	}
 }
 
-.components-button.pressable-overview-plan-selection__filter-button {
+.theme-a8c-for-agencies .components-button.pressable-overview-plan-selection__filter-button {
```

Comment:
Better targeting to keep the styles applied after the page is refreshed.

Reply:
👍